CRM/Core/Resources.php

```diff
@@ -868,7 +868,7 @@ public static function renderMenubarStylesheet(GenericHookEvent $e) {
       '$breakMin' => $params['breakpoint'] . 'px',
       '$breakMax' => ($params['breakpoint'] - 1) . 'px',
       '$menubarColor' => $menubarColor,
-      '$menuItemColor' => $params['menuItemColor'] ?? 'rgba(' . implode(', ', CRM_Utils_Color::getRgb($menubarColor)) . ", .9)",
+      '$menuItemColor' => $params['menuItemColor'] ?? 'rgb(' . implode(', ', CRM_Utils_Color::getRgb($menubarColor)) . ")",
```

Comment:
The only reason for this conversion was to add the `rgba` opacity info. So omitting that we can simply do

```suggestion
      '$menuItemColor' => $params['menuItemColor'] ?? $menubarColor,
```